.github/workflows/performance.yml

```diff
@@ -76,13 +76,14 @@ jobs:
             - name: Compare performance with base branch
               if: github.event_name == 'push'
               # The base hash used here need to be a commit that is compatible with the current WP version
-              # The current one is c7722262e65a3f4d0f1a2d1ad29eccb2069509e4 and it needs to be updated every WP major release.
+              # The current one is 177c166a5eada898ad015f5efaf1c6fefa7042c4 and it needs to be updated every WP major release.
```

Comment:
I think I need to use a different SHA; this doesn't work as expected, and I don't see new commits in CodeVitals.